RampUpAuto/src/main/java/seleautomation/com/RampUpAuto/GoogleSearch.java

```diff
@@ -0,0 +1,47 @@
+package Testing;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import java.util.concurrent.TimeUnit;
+
+   public class TestTest  {
+WebDriver driver;
+@BeforeMethod
+   public void setup()  {
+
+
+    System.setProperty("webdriver.chrome.driver", "C:\\Users\\srabaya\\IdeaProjects\\Selenium\\chromedriver.exe");
+    driver = new ChromeDriver();
+    driver.get("https://www.google.com");
+    driver.manage().window().maximize();
+
+}
+@Test
+   public void GoogleSearchTC() throws InterruptedException {
+
+    driver.manage().timeouts().implicitlyWait(30, TimeUnit.SECONDS);
+    driver.findElement(By.name("q")).sendKeys("Selenium Tutorials");
+    driver.findElement(By.xpath("//input[@name='btnK']")).click();
+    Thread.sleep(10000);
```

Comment:
Try to remove it please , no need of it, as we always trying to limit the time delays in the code as possible as we can